packages/nextjs/app/builders/0xF058bE573Fa9303095B9bdED77EE2867d75BbFa6/page.tsx

```diff
@@ -0,0 +1,131 @@
+import React from "react";
+import Image from "next/image";
+import { NextPage } from "next";
+import { FaGithub, FaTelegram, FaTwitter } from "react-icons/fa";
+import { MotionDiv, MotionS } from "~~/components/MotionElements";
+import { Address } from "~~/components/scaffold-eth";
+
+const PortfolioPage: NextPage = () => {
+  const socials = [
+    { name: "GitHub", url: "https://github.com/amsorrytola", icon: <FaGithub className="w-5 h-5" /> },
+    { name: "Twitter", url: "https://x.com/An92147600_", icon: <FaTwitter className="w-5 h-5" /> },
+    { name: "Telegram", url: "https://t.me/tolaamsorry", icon: <FaTelegram className="w-5 h-5" /> },
+  ];
+
+  const skills = [
+    { name: "Solidity", icon: "🛠️" },
+    { name: "Foundry", icon: "🧪" },
+    { name: "TypeScript", icon: "🔷" },
+    { name: "React", icon: "⚛️" },
+    { name: "Next.js", icon: "⏭️" },
+    { name: "Tailwind CSS", icon: "💨" },
+    { name: "Wagmi & RainbowKit", icon: "🌈" },
+    { name: "zkEmail", icon: "🔐" },
+    { name: "Bitcoin + Runes Protocol", icon: "₿" },
+    { name: "API & ERP Integration", icon: "🔗" },
+    { name: "Smart Contract Events & Hooks", icon: "📚" },
+  ];
+
+  const projects = [
+    {
+      title: "Bitcoin Etching API",
+      stack: "Node.js • Bitcoin Core • Testnet",
+      description:
+        "Built a custom Bitcoin API for etching Runes using regtest and migrated to testnet with UniSat Wallet support.",
+    },
+    {
+      title: "zkEmail Attendance dApp",
+      stack: "zkEmail • Solidity • Scaffold-ETH 2",
+      description: "Decentralized check-in system using zk proofs of email, integrated with BatchRegistry on Arbitrum.",
+    },
+    {
+      title: "ERP Integration Suite",
+      stack: "Node.js • REST APIs • MongoDB",
+      description: "Built a backend to unify third-party ERP systems with vendor-normalized data and secure endpoints.",
+    },
+  ];
```

Comment:
Since these are all constant, it makes sense to place them outside of the component. So they're created only once at module initialization, rather than every render.